### ISSUE TITLE
postinstall step failing on CircleCI

### ISSUE THREAD

**Issue by theolampert** [joshswan/react-native-globalize#48]
Using yarn on circleci version 4.0.0 of the package fails to build, giving the following output:

```
[4/4] Building fresh packages...
error /home/circleci/project/node_modules/react-native-globalize: Command failed.
Exit code: 1
Command: patch-package
Arguments: 
Directory: /home/circleci/project/node_modules/react-native-globalize
Output:
patch-package 6.2.0
Applying patches...
Error: Patch file found for package globalize which is not present at node_modules/@types/globalize
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

**Comment by joshswan** [comment]
Remove the postinstall entirely so should be fixed on `v4.0.1`

**Comment by theolampert** [comment]
@joshswan thank you!
